osu.Game/Screens/OsuScreen.cs

```diff
@@ -138,6 +170,8 @@ public override void OnEntering(IScreen last)
 
             backgroundStack?.Push(localBackground = CreateBackground());
 
+            ScreenActivity = InitialScreenActivity;
```

Comment:
If you are doing this in OnEntering, you likely want to protect against a screen calling `ScreenActivity_Set` before this happens. Or handle this case (`if (ScreenActivity == null) { ScreenActivity = InitialScreenActivity; }`) and document such.